server/api/__init__.py

```diff
@@ -20,25 +20,6 @@
 rest_api.init_app(app)
 CORS(app)
 
-
-# Setup database
-@app.before_first_request
```

Comment:
This function `@app.before_first_request` is deprecated in the most recent flask